apps/video/js/video.js

```diff
@@ -108,6 +108,9 @@ var loadingChecker =
   new VideoLoadingChecker(dom.player, dom.inUseOverlay, dom.inUseOverlayTitle,
                           dom.inUseOverlayText);
 
+// Store seek target before previous one is done.
+var pendingSeekPosition = NaN;
+
```

Comment:
I'd use null here rather than NaN.  null seems more idiomatic JS, since we don't have to worry about type safety.  It could be that NaN is more optimizable, I suppose.